src/components/mdSwitch/mdSwitch.scss

```diff
@@ -11,6 +11,10 @@ $switch-ripple-size: 48px;
   display: inline-flex;
   position: relative;
 
+  &.md-checked .md-switch-container .md-switch-thumb {
```

Comment:
This code was on the Javascript for one reason only: Later I will develop this component with touch gestures. So I need to calculate the distance using Javascript. I liked the idea to move this to css and switch classes after the drag event.